index.html

```diff
@@ -12185,6 +12186,67 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="property" id="aria-repeated">
+			<pdef>aria-repeated</pdef>
+			<div class="property-description">
+				<p>Indicates content is repeated for visual purposes.</p>
+				<p>The <code>aria-repeated</code> helps indicate that content is repeated, and that presenting the repeated content to assistive technology users could produce underesirable redundancy in the output. Unlike <pref>aria-hidden</pref>, <code>aria-repeated</code> content may be presented by screen readers to users when requested by the user.</p>
+				<p>Authors SHOULD NOT use <code>aria-repeated</code> on the very first instance of the repeated content, and SHOULD use <code>aria-repeated</code> on subsequent redundant instances of the same content. Setting <code>aria-repeated</code> on a subtree indicates the entire subtree is redundant, and it cannot be reversed within that subtree by setting <code>aria-repeated="false"</code> on a descendant.</p>
```

Comment:
What if the very first instance of the repeated content is a pull quote which, for aesthetic reasons, appears first?

In addition, I assume that authors MAY use <code>aria-repeated</code> on the very first instance of the repeated content -- IF they set it to `false`.

Therefore what about something in the ballpark of: 

      Authors SHOULD only set <code>aria-repeated</code> to <code>true</code>
      on redundant instances of content because assistive technologies MAY skip
      over these instances during presentation to the user.

Reply:
Ok, I took this but wanted to keep the part about aria-repeated=false.
Could we actually disallow the false value completely? I don't see how it's useful.

Reply:
I'm good with this change, but I'm not resolving the conversation yet because you ask about disallowing the `false` value completely. That might be a discussion worth having with the group.

I personally think disallowing `false` on a boolean would be weird. So either `false` is the default value that no one uses because why bother? Or `aria-repeated` shouldn't be a boolean. IMHO.

Reply:
If I have a submit button at the beginning and one at the end of a form, wouldn't it be better to use `aria-repeated=true` on the first submit button?

Reply:
Probably should tell authors not to apply it to elements that contain interactive content. I would not want my screen reader to hide either submit button regardless of what repeated is set to.